integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -258,3 +258,9 @@
 0.6.40
 
 - Fix incorrect directory when mounting & formatting master journal volume
+
+0.6.41
+
+- Remove javaOptions under CSI
```

Comment:
Why remove `javaOptions`?

Reply:
I was thinking about putting all properties in a central place (the `properties` section). In your case, do different clients have different properties?